pkg/registry/deployment.go

```diff
@@ -71,6 +73,16 @@ func GenerateDeployment(cr *registryv1alpha1.DevfileRegistry, scheme *runtime.Sc
 									},
 								},
 							},
+							Env: []corev1.EnvVar{
+								{
+									Name:  "ENABLE_TELEMETRY",
+									Value: strconv.FormatBool(*cr.Spec.Telemetry.Enabled),
+								},
+								{
+									Name:  "REGISTRY",
```

Comment:
Could we use a more descriptive environment variable than `REGISTRY`?

Reply:
I will update it to use `REGISTRY_NAME`